wallet-new/src/Cardano/Wallet/WalletLayer/Kernel/Conv.hs

```diff
@@ -193,9 +193,15 @@ toAddress acc hdAddress =
     V1.WalletAddress (V1 cardanoAddress)
                      (addressMeta ^. addressMetaIsUsed)
                      (addressMeta ^. addressMetaIsChange)
+                     (V1 addressOwnership)
   where
-    cardanoAddress = hdAddress ^. HD.hdAddressAddress . fromDb
-    addressMeta    = acc ^. HD.hdAccountState . HD.hdAccountStateCurrent . cpAddressMeta cardanoAddress
+    cardanoAddress   = hdAddress ^. HD.hdAddressAddress . fromDb
+    addressMeta      = acc ^. HD.hdAccountState . HD.hdAccountStateCurrent . cpAddressMeta cardanoAddress
+    addressOwnership =
```

Comment:
Me and @KtorZ talked about this and we realised this is unnecessary complication: if we are converting from a kernel type (which remember, it's stored in our `DB` to begin with), surely this is our address, because otherwise we wouldn't be able to construct an `HDAddress` without having the derivation index of the parent account etc. This implies ownership, therefore here it's simply `V1.AddressIsOurs`.

It's a good idea to bang a comment in the code as well to justify our rationale.

Reply:
Amended with what was discussed above :+1: